extensions/wikia/Search/classes/Result.php

```diff
@@ -3,33 +3,38 @@
  * Class definition for Wikia\Search\Result
  */
 namespace Wikia\Search;
+
 use \Solarium_Document_ReadWrite as ReadWrite; #forward compatibility with v3
 /**
  * This is a wrapper for the Solarium_Document_ReadWrite class based on code we
  * originally wrote for an entirely hand-rolled search result class.
+ *
  * @author Robert Elwell
  * @package Search
  */
 class Result extends ReadWrite {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Class "Result" has 25 methods, which is greater than 20 authorized. Split it into smaller classes. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)